src/lib/Tests/Form/DataMapper/PolicyCreateMapperTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Tests\Form\DataMapper;
+
+use eZ\Publish\Core\Repository\Values\User\PolicyCreateStruct;
+use eZ\Publish\API\Repository\Values\Content\LocationCreateStruct;
+use EzSystems\EzPlatformAdminUi\Form\Data\Policy\PolicyCreateData;
+use EzSystems\EzPlatformAdminUi\Form\Data\Policy\PolicyUpdateData;
+use EzSystems\EzPlatformAdminUi\Form\DataMapper\PolicyCreateMapper;
+use PHPUnit\Framework\TestCase;
+use EzSystems\EzPlatformAdminUi\Exception\InvalidArgumentException;
+
+class PolicyCreateMapperTest extends TestCase
+{
+    /**
+     * @var PolicyCreateMapper
+     */
+    private $mapper;
+
+    public function setUp()
+    {
+        $this->mapper = new PolicyCreateMapper();
+    }
+
+    public function tearDown()
+    {
+        unset($this->mapper);
+    }
+
+    /**
+     * @dataProvider dataProvider
+     *
+     * @param array $properties
+     */
+    public function testMap(array $properties)
+    {
+        $data = $this->mapper->map($this->createStruct($properties));
+
+        $this->assertEquals($this->createData($properties), $data);
+    }
+
+    /**
+     * @dataProvider dataProvider
+     *
+     * @param array $properties
+     */
+    public function testReverseMap(array $properties)
+    {
+        $struct = $this->mapper->reverseMap($this->createData($properties));
+
+        $this->assertEquals($this->createStruct($properties), $struct);
+    }
+
+    public function testMapWithWrongInstance()
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('Argument \'value\' is invalid: must be an instance of ' . PolicyCreateStruct::class);
```

Comment:
Fine for me but `sprintf()` looks better instead of string concatenation. But this can stay as-is.